janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/optimize/strategy/JanusGraphStepStrategy.java

```diff
@@ -63,17 +62,19 @@ public void apply(final Traversal.Admin<?, ?> traversal) {
             } else {
                 //Make sure that any provided "start" elements are instantiated in the current transaction
                 final Object[] ids = originalGraphStep.getIds();
-                ElementUtils.verifyArgsMustBeEitherIdOrElement(ids);
```

Comment:
Was this removed because of [TINKERPOP-2507](https://issues.apache.org/jira/browse/TINKERPOP-2507)?